Makefile

```diff
@@ -223,8 +223,8 @@ build-push-image: $(CONFIG_DOCKER_TARGET) $(CONFIG_DOCKER_TARGET_QUAY) build-ope
 	@docker push $(QUAY_REGISTRY)/$(OPERATOR_IMAGE_NAME)-$(LOCAL_ARCH):$(VERSION)
```

Comment:
Should comment out this line and [line 221](https://github.com/IBM/ibm-common-service-operator/pull/436/files/c8cdb8bdb8ddc67e3538559120332aa4a237f226..6f2ad30fc3a9d1abeeca96df021ca9c51314d8d5#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L221) as well. No need for PROW to push to quay at all

Reply:
Actually I don't think this is the problem. 
For example, it pushes to repo `ibm-common-service-operator-amd64`, not the repo `ibm-common-service-operator` with `multiarch`.

and anyhow we should avoid to push image to `quay.io`